Week1/homework/nadin.js

```diff
@@ -0,0 +1,111 @@
+'use strict';
+let greetingIndonesian = 'Ciao';
+let greetingArabic = 'مرحبا ';
+let greetingSpanish = 'Halo';
+ console.log(greetingIndonesian, greetingArabic, greetingSpanish);
+
+
+console.log("I'm awesome");
+
+
+let x;
+console.log("The value of my variable number will be: undefined");
+console.log(x);
+x = 1;
+console.log("The value of my variable number will be: 1");
+console.log(x);
+
+let y = "the first string";
+console.log("the value of my variable y will be: the first string");
+console.log(y);
+y = "the second string";
+console.log("the value of my variable string1 will be: the second string");
+console.log(y);
+
+console.log(Math.round(7.25));
+// expected output: 8
+
+let z = 7.25;
+console.log(z);
+
+let a = 8;
+console.log(a);
+
+console.log(z !== a);
+console.log(z == a);
+console.log(z <= a);
+
+let b = 8;
+console.log(b);
+
+let colors = [];
+console.log("the value of my variable y will be: an empty array");
+console.log(colors);
+
+let animals = ['dog', 'cat', 'horse', 'fish', 'bird'];
```

Comment:
You can use `const` here for both `colors` and `animals` variable since we aren't reassigning these two variables anywhere 🙂